### PR TITLE
Added Bitcoin Receiver & Transaction objects

### PR DESCRIPTION
**Pull request by maxwells**

### CODE REVIEW

**Comment by maxwells** [comment]
hey Brian -- just as a heads up, I've updated this PR by removing BitcoinTransaction as a top level API object

File: lib/Stripe/BitcoinTransaction.php [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+class Stripe_BitcoinTransaction extends Stripe_ApiResource
```
Comment (bkrausz) [inline lib/Stripe/BitcoinTransaction.php]: Is the intent here that bitcoin transactions will properly have their own object, but you won't be able to do anything special with them like save?

Comment (maxwells) [inline lib/Stripe/BitcoinTransaction.php]: Yes. They're only createable through Customer action (eg. transferring funds to the receiver). I'm certainly open to having the BitcoinTransaction sublist on BitcoinReceiver just be associative arrays, though

Comment (bkrausz) [inline lib/Stripe/BitcoinTransaction.php]: They do seem to be listable and fetchable, so do we want to have `retrieve` and `all` methods?

Comment (maxwells) [inline lib/Stripe/BitcoinTransaction.php]: I chatted about it with @raycmorgan and @canderson and we felt it was an unnecessarily granular unit to expose at the bindings level. It represents the set of transactions that fill a bitcoin receiver (customers pushing coins to the receiver) and we couldn't think of much of a valid use case for querying those outside the context of a bitcoin receiver. The proposed changes here does allow them to be pulled as a sublist

```php
$receiver->transactions->all();
```

How do you feel about this approach?

Comment (bkrausz) [inline lib/Stripe/BitcoinTransaction.php]: works for me, thanks for clarifying. :sheep:

**Pull request merged**